app/views/includes/forms/errorSummary.scala.html

```diff
@@ -0,0 +1,43 @@
+@*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(
+    heading: String,
+    form: Form[_],
+    classes: Seq[String] = Seq.empty,
+    dataJourney: Option[String] = None
+)(implicit messages: Messages)
+
+<div class="error-summary@if(form.hasErrors) { error-summary--show} @classes.mkString(" ")"
+    id="error-summary-display"
+    role="alert"
+    aria-labelledby="error-summary-heading"
+    tabindex="-1">
+    <h2 id="error-summary-heading" class="heading-medium error-summary-heading">@heading</h2>
+    <ul class="error-summary-list">
+    @if(form.hasErrors) {
+        @form.errors.map { error =>
+            <li role="tooltip" @dataJourney.map(page => Html(s"""data-journey="${page}:error:${error.key}""""))>
```

Comment:
I need to look into the error detail with regards to stageprompt GA.
Paul Cook pointed out that recent changes (probably radio changes) have broken our GA with regard to setting the 'error - field' event category on a failed submit. I haven't had a chance to poke around yet though.
Looks like we don't get the data-journey attribute set at the moment (on the WDYWTD page at least)

Reply:
Yeah, sounds like a separate ticket